packages/gestalt/src/Masonry/defaultLayout.js

```diff
@@ -1,7 +1,13 @@
 // @flow strict
 import { type Cache } from './Cache.js';
```

Comment:
i was thinking about this and we should probably fork this into a separate layout (at least until after the experiment):
- makes it easier to run the experiment (i.e. if experiment + two column, then use defaultWithTwoColumnLayout)
- ensures we don't make any unexpected changes to the existing default Layout

Reply:
Yeah, that makes sense. I've been thinking about if this should rely on just the presence of 2-col modules or if there should also be an experiment prop…the experiment prop isn't technically necessary but probably safer, and does make forking this layout easier.